cmd/server/sqlite.go

```diff
@@ -28,7 +28,7 @@ var (
 		`create table if not exists company_watches(id primary key, company_id, webhook, auth_token, created_at datetime, deleted_at datetime);`,
 
 		// OFAC download stats
-		`create table if not exists ofac_download_stats(downloaded_at datetime, sdns, alt_names, addresses);`,
+		`create table if not exists ofac_download_stats(downloaded_at datetime, sdns, alt_names, addresses, denied_persons);`,
```

Comment:
We will need to support this as a migration (`alter table add colulm ...`) instead of on the `create table` call.

Reply:
We can skip this for now, but for the next release I'm going to implement proper migrations.